src/StatusQ/Components/StatusListItem.qml

```diff
@@ -1,4 +1,6 @@
 import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
```

Comment:
If you do't mind pls use minor version of QtQuick import

Reply:
You're referring to Controls and Layouts? because QtQuick was there already